src/projectM-emscripten/Makefile.am

```diff
@@ -0,0 +1,18 @@
+AM_CPPFLAGS = \
+${my_CFLAGS} \
+-include $(top_builddir)/config.h \
+-DSYSCONFDIR=\""$(sysconfdir)"\" \
+-I${top_srcdir}/src/libprojectM \
+-I${top_srcdir}/src/libprojectM/Renderer \
+USE_SDL=2 \
+${SDL_CFLAGS}
+
+REALSRCDIR=${top_srcdir}/src/projectM-sdl
+
+bin_PROGRAMS = projectW.bc
+projectW_bc_SOURCES = $(REALSRCDIR)/pmSDL.cpp $(REALSRCDIR)/projectM_SDL_main.cpp $(REALSRCDIR)/pmSDL.hpp
```

Comment:
projectM_SDL_emscripten.cpp should be deleted ?

Reply:
Yeah most likely true. `projectM_SDL_emscripten.cpp` is from long long ago. Will need to look into this at some point.